website/package.json

```diff
@@ -24,6 +24,7 @@
     "react-device-detect": "^2.2.3",
     "react-dom": "^18.2.0",
     "react-helmet": "^6.1.0",
+    "react-icons": "^4.10.1",
```

Comment:
Kindly **uninstall** this icon library to avoid packing up the app. The website already uses **@mui/icons-material** library so no need for the react-icons.
Just visit the **mui icons** [website](https://mui.com/material-ui/material-icons/) to search for icons you want to use then import them into your file accordingly.
Example: https://mui.com/material-ui/material-icons/?query=new+&selected=OpenInNew